sdk/communication/Azure.Communication.CallingServer/src/Models/Events/AcsEventType.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallingServer
+{
+    public partial struct AcsEventType
```

Comment:
This is overwriting generated AcsEventType right? it is difficult to tell which is overwriting, and which is standalone model. Can you add [CodeGenModel("??")] for all overwriting models?

Reply:
added!